src/types/AuditLogEvent.ts

```diff
@@ -0,0 +1,4 @@
+export default interface AuditLogEvent {
```

Comment:
We might need to make the definition of an event match the ones we store in the Audit Log API but we could do this at a later date

Reply:
Agreed. I left it for the next PR when we are adding the existing audit logs to Core.